extensions/HuggingFace/python/pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = {file = ["requirements.txt"]}
 [project.urls]
 Homepage = "https://github.com/lastmile-ai/aiconfig/"
 Issues = "https://github.com/lastmile-ai/aiconfig/issues"
+LastMile = "https://lastmileai.dev/"
```

Comment:
nit, is this needed?

Reply:
not needed but thought it might be nice to add. shoud I remove it

Reply:
I don't feel strongly about it, just think it's not really related to the extension

Reply:
fair enough. Removed it, can be updated next time we publish